test/server/cards/01_SOR/VadersLightsaber.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('Vader\'s Lightsaber', function() {
+    integration(function() {
+        describe('Vader\'s Lightsaber\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['vaders-lightsaber'],
+                        groundArena: ['darth-vader#commanding-the-first-legion'],
+                        spaceArena: ['tieln-fighter'],
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        spaceArena: ['cartel-spacer'],
+                    }
+                });
+            });
+
+            it('should deal 4 damage to a ground unit when attached to the Darth Vader unit', function () {
+                this.player1.clickCard(this.vadersLightsaber);
+                expect(this.player1).toBeAbleToSelectExactly([this.darthVader, this.wampa]);    // cannot attach to vehicles
+                this.player1.clickCard(this.darthVader);
+
+                expect(this.darthVader).toHaveExactUpgradeNames(['vaders-lightsaber']);
+                expect(this.player1).toBeAbleToSelectExactly([this.wampa, this.darthVader]);
+                expect(this.player1).not.toHavePassAbilityPrompt();
```

Comment:
The when played for Vader's Lightsaber is a "may" ability. Am I misreading or does this line contradict that?